tests/acceptance/visual-regression-test.js

```diff
@@ -30,6 +30,10 @@ module('Acceptance | visual regression', function(hooks) {
       { route: '/mascots/faq/', title: '/tomster/faq.html' },
       { route: '/mascots/commission/', title: '/tomster/commission/index.html' },
       { route: '/statusboard', title: '/statusboard.html' },
+      { route: '/releases', title: '/releases.html' },
+      { route: '/releases/release', title: '/releases/release.html' },
+      { route: '/releases/canary', title: '/releases/canary.html' },
+      { route: '/releases/beta', title: '/releases/beta.html' },
     ]
 
     await pages.reduce(async (prev, config) => {
```

Comment:
Wow. This is a really nice way to watch for visual regressions. I will have to put this in my pocket for future projects.

Reply:
Oh just you wait! 😂 I have a whole process that I'm going to be doing with the percy tests comparing the old legacy website with this new one! I will probably blog about it too 😉